### DISCUSSION TITLE
[Feature request] auto import in typescript repo and relative import feature

### DISCUSSION THREAD

**Discussion by ztlevi** [joaotavora/eglot#995] [Ideas]
Hi, I try to find some interactive function I can call to import the symbol but couldn't. Also, once the symbol is auto completed, it should be able to import the symbol correctly but I need to run `eglot-code-action-quickfix` to do the import.

I found auto import in typescript repo does not work but in python repo it does works. 

In typescript repo, I use typescript-language-server, in python repo, I use pyright.

I wonder if the auto import is a missing feature for typescript-language-server. Hence requested here.

Also, I found vscode has this setting for import. I didn't find any similar setting in eglot. Is it something also missing?
```
  "typescript.preferences.importModuleSpecifier": "relative",
```

Thanks!

**Comment by ztlevi** [comment]
This config in lsp-mode which works for me. 
```
(setq lsp-clients-typescript-init-opts
      '(:importModuleSpecifierPreference "relative"))
```

Is there any equivalent in eglot? 

Thanks!

**Comment by jeff-phil** [comment]
This is probably what you're looking for:

https://www.reddit.com/r/emacs/comments/11bqzvk/emacs29_and_eglot_inlay_hints/
